src/main/java/web/pages/flipkart/SearchPage.java

```diff
@@ -0,0 +1,72 @@
+package web.pages.flipkart;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Iterator;
+import static com.codeborne.selenide.Selenide.$$x;
+import static com.codeborne.selenide.Selenide.$x;
+
+public class SearchPage {
+    ArrayList a1 = new ArrayList();
+    ArrayList a2 = new ArrayList();
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+    public SearchPage selectPageNumber(int no) {
+        $x("//a[@class='_2Xp0TH'][contains(text(),'"+no+"')]").shouldBe(Condition.visible).click();
+        logger.info("second page price selected");
+        return this;
+    }
+
+    /**
+     * countingPagePrices: performing pagination
+     * @return All prices of page1 & page2
+     */
+    public SearchPage countingPagePrices() {
+        $x("//div[@class='_1vC4OE']").shouldBe(Condition.visible);
+        ElementsCollection priceList = $$x("//div[@class='_1vC4OE']");
+        Iterator<SelenideElement> iterate_priceList = priceList.iterator();
+        while(iterate_priceList.hasNext()){
+            SelenideElement tempPriceList = iterate_priceList.next();
+            String singlePriceList = tempPriceList.getText();
+            String price_without_rupees = singlePriceList.split("\u20B9")[1];
```

Comment:
Can refactor this by :
String price_without_rupees = iterate_priceList.next().getText().split("\u20B9")[1];